x-pack/legacy/plugins/apm/e2e/ci/kibana.e2e.yml

```
@@ -0,0 +1,31 @@
+# Kibana
+server.port: 5701
+xpack.security.encryptionKey: 'something_at_least_32_characters'
+csp.strict: false
+logging.verbose: true
+
+# Elasticsearch
+# Started via apm-integration-testing
+# ./scripts/compose.py start master --no-kibana --elasticsearch-port 9201 --apm-server-port 8201
+elasticsearch.hosts: http://localhost:9201
+elasticsearch.username: 'kibana_system_user'
```

`kibana_system_user` didn't work when I tried, so maybe `admin` could be a good one, what do you think?
```suggestion
elasticsearch.username: 'admin'
```

---

Kibana shouldn't connect to ES with superuser privileges. According to [the docs](https://github.com/elastic/apm-integration-testing#logging-in) `kibana_system_user` should exist.

I can confirm that `kibana_system_user` can access ES like:

```
curl --user kibana_system_user:changeme -X GET "localhost:9201/_security/user/_has_privileges?pretty" -H 'Content-Type: application/json' -d'{ "cluster": [ "monitor", "manage" ] }'
```

which returns:
```json
{
  "username" : "kibana_system_user",
  "has_all_requested" : false,
  "cluster" : {
    "monitor" : true,
    "manage" : false
  },
  "index" : { },
  "application" : { }
}
```

---

Can you try running that and LMK if it works?